pyvista/utilities/helpers.py

```diff
@@ -443,6 +444,11 @@ def vtk_points(points, deep=True):
         Perform a deep copy of the array.  Only applicable if
         ``points`` is a :class:`numpy.ndarray`.
 
+    force_float : bool, optional
+        Casts the datatype to float32 if points datatype are
+        non-float.  Set this to ``False`` to allow non-float types,
+        though this may lead to errors when transforming datasets.
```

Comment:
```suggestion
        Casts the datatype to ``float32`` if points datatype is
        non-float.  Set this to ``False`` to allow non-float types,
        though this may lead to bugs when transforming datasets.
```
"Errors" sounds like raising an exception, but the risk is a silently wrong result. I'm open to different phrasings here, it's just the word "error" I'd prefer to avoid.

Reply:
Agreed. I modified your suggestion a bit.